tests/test_feeds/episode.rss

```diff
@@ -151,6 +151,33 @@ Play CORVETTE TODAY on your smart device too! Just say, "Hey Alexa (or hey Googl
             <adori:interactiveTagCount>0</adori:interactiveTagCount>
             <ihr:interactiveTagCount>0</ihr:interactiveTagCount>
         </item>
+         <item>
+            <title>CORVETTE TODAY #111 - Corvette News &amp; Headlines, Early June 2020</title>
+            <link>https://podcasts.adorilabs.com/show/e?eid=ID6aGhYYXWHO7v5v</link>
+            <author>Steve Garrett</author>
+            <itunes:author>Steve Garrett</itunes:author>
+            <content:encoded>test</content:encoded>
+            <description>description</description>
+            <itunes:summary><![CDATA[<div><span >It's the unofficial start to Summer and CORVETTE TODAY is shining brightly with news and headlines with Keith Cornett from </span><a href="http://corvetteblogger.com/" target="_blank" rel="noopener noreferrer">CorvetteBlogger.com</a><span >.</span><br></div><div><br></div><p>It seems as though there never a shortage of Corvette information buzzing around.  And this week is no exception.&nbsp;</p><p><br></p><p>&nbsp;Here are a few of the topics covered in this show…..</p><div><p><br></p><p>	1.     Corvette raises destination charge by $100 to $1,395</p><p>	2.     Chevrolet adds 2 new videos to C8 Z06 Academy</p><p></p><p>	3.     NCM names Brian Baker as Director of Collections &amp; Education</p><p>	4.  Trifecta Performance claims to have cracked the ECM on the C8</p><p></p><p>	5.     The online 2023 Z06 Owner's Manual confirms the Z06 to go 200 mph</p><p>	6.     Corvette brings home the eNVy award for best premium sports car</p><p></p><p>	7.     Chevrolet not going to SEMA this year</p><p>	8.     Collection of Indy 500 Pace Cars sells for $1.375m at Mecum Indy.</p><div><br></div><div>Start your Summer off right with CORVETTE TODAY on podcast and YouTube!</div><div><br></div><div><br></div><div><b>Listen to the show, watch the YouTube video, visit the website, sign up for CORVETTE TODAY email notifications&nbsp;</b><b >and join the Facebook group at:</b></div><div><br></div><div><br></div><div><a href="http://www.corvettetodaypodcast.com/" target="_blank" rel="noopener noreferrer">www.CorvetteTodayPodcast.com</a></div></div>]]></itunes:summary>
+            <itunes:subtitle>News &amp; Headlines, Early June 2022</itunes:subtitle>
+            <itunes:owner>
+                <itunes:name>Steve Garrett</itunes:name>
+                <itunes:email>SteveGarrettDJ@gmail.com</itunes:email>
+            </itunes:owner>
+            <itunes:duration>2785</itunes:duration>
+            <guid>adori-8d2abc8f-65a4-401f-a6ef-45d86a24e0be</guid>
+            <pubDate>Mon, 22 May 2023 14:00:00 +1000</pubDate>
```

Comment:
do we not need AEST at the end of the date to identify this?

Reply:
+1000 means AEST